### PR TITLE
more docs of advanced search 

### LINKED ISSUE #99: How does AND work in search
I.e. I'd expect to get 66 hits for the search `pkgsearch::advanced_search(VignetteBuilder = "knitr AND rmarkdown")` like when following [Julia Silge's blog post](https://juliasilge.com/blog/mining-cran-description/)

``` r
cran <- tools::CRAN_package_db()

# the returned data frame has two columns with the same name???
cran <- cran[,-65]

# make it a tibble
cran <- tibble::as_tibble(cran)

dplyr::count(cran, VignetteBuilder, sort = TRUE)
#> # A tibble: 29 x 2
#>    VignetteBuilder             n
#>    <chr>                   <int>
#>  1 <NA>                    11101
#>  2 knitr                    4279
#>  3 R.rsp                     198
#>  4 knitr, rmarkdown           46
#>  5 knitr, R.rsp               38
#>  6 utils, knitr               10
#>  7 R.rsp, knitr                9
#>  8 rmarkdown, knitr            9
#>  9 utils                       9
#> 10 knitr, rmarkdown, R.rsp     6
#> # … with 19 more rows
dplyr::filter(
  cran,
  grepl("rmarkdown", VignetteBuilder),
  grepl("knitr", VignetteBuilder)
)$Package
#>  [1] "AMR"                   "BIGL"                  "ciTools"              
#>  [4] "CoopGame"              "cyphr"                 "data.tree"            
#>  [7] "denoiSeq"              "drugCombo"             "eikosograms"          
#> [10] "ergm"                  "fitHeavyTail"          "gasfluxes"            
#> [13] "GFD"                   "ggTimeSeries"          "gprofiler2"           
#> [16] "grainscape"            "idiogramFISH"          "imputeFin"            
#> [19] "LTRCtrees"             "MANOVA.RM"             "MiRKAT"               
#> [22] "NetLogoR"              "nlpred"                "odr"                  
#> [25] "optband"               "PCMBase"               "PCMBaseCpp"           
#> [28] "PMCMRplus"             "polyCub"               "polyRAD"              
#> [31] "portfolioBacktest"     "POUMM"                 "ppclust"              
#> [34] "prettydoc"             "QFASA"                 "qualvar"              
#> [37] "quickPlot"             "radiomics"             "RcppNumerical"        
#> [40] "reproducible"          "riskParityPortfolio"   "robvis"               
#> [43] "RSpectra"              "SeqKat"                "sequenza"             
#> [46] "showtext"              "sim1000G"              "SpaDES"               
#> [49] "SpaDES.addins"         "SpaDES.core"           "sparseIndexTracking"  
#> [52] "spathial"              "spectralGraphTopology" "SSDM"                 
#> [55] "stylest"               "SubgrPlots"            "supc"                 
#> [58] "SuperLearner"          "sweidnumbr"            "tergm"                
#> [61] "utf8"                  "VCA"                   "VFP"                  
#> [64] "visNetwork"            "wrProteo"              "zooaRch"

pkgsearch::advanced_search(VignetteBuilder = "knitr AND rmarkdown")
#> - "advanced search" -------------------------- 3812 packages in 0.007 seconds -
#>   #     package        version by                         @ title              
#>   1 100 tufterhandout  1.2.1   Michael C Sachs           5y Tufte-style html...
#>   2  98 downloadthis   0.2.0   Felipe Mattioni Maturana 28d Implement Downlo...
#>   3  91 posterdown     1.0     Brent Thorne              8M Generate PDF Con...
#>   4  87 knitrBootstrap 1.0.2   Jim Hester                2y 'knitr' Bootstra...
#>   5  87 texPreview     1.4.4   Jonathan Sidi             4M Compile and Prev...
#>   6  87 rmarkdown      2.2     Yihui Xie                 2d Dynamic Document...
#>   7  86 linl           0.0.4   Dirk Eddelbuettel         7M 'linl' is not 'L...
#>   8  85 details        0.2.1   Jonathan Sidi             5M Create Details H...
#>   9  85 rjdmarkdown    0.1.0   Alain Quartier-la-Tente  24d 'rmarkdown' Exte...
#>  10  83 BDP2           0.1.3   Manuel Wiesenfarth        2y Bayesian Adaptiv...
```

<sup>Created on 2020-06-02 by the [reprex package](https://reprex.tidyverse.org) (v0.3.0)</sup>

Cf https://github.com/r-lib/usethis/issues/1147

### PR DESCRIPTION
**Pull request by maelle**
Fix #99

### CODE REVIEW

File: R/advanced_search.R [hunk 1]
```diff
@@ -20,7 +20,14 @@
 #' # All orphaned packages
 #' advanced_search(Maintainer = "ORPHANED")
 #'
-#' # Packages that match a certain field
+#' # Packages with both Hester and Wickham as authors
+#' advanced_search(Author = "Hester", Author = "Wickham")
+#' advanced_search("Author: Hester AND Author: Wickham")
+#' 
+#' # Packages with Hester as an Author, and Wickham in any field
+#' advanced_search(Author = "Hester AND Wickham")
```
Comment (gaborcsardi) [inline R/advanced_search.R:28]: I think we could just leave this one out? It is confusing imo. Maybe write it as

```r
advanced_search(Author = "Hester", "Wickham")
```
and the one below as 

```r
advanced_search(Author = "Hester", "NOT Wickham")
```

and also add this example:

```r
advanced_search(Author = "Hester", Author = "NOT Wickham")
```
?

**Comment by maelle** [comment]
more comments @gaborcsardi? or should we just close this?

**Comment by gaborcsardi** [comment]
Oh, I did not realize you updated the one thing I commented on.

**Comment by maelle** [comment]
my fault for not pining you when I did!

**Pull request merged**